src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -769,7 +769,7 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
                 { this.props.showFilter ? (
                     <div className={`pull-right form-group has-feedback input-group-sm tableFilter`} style={{ display:'inline-block', marginLeft: 5}}>
                         <input ref={this.handlers.filterInputRef} placeholder={this.props.filterPlaceholder || ""} type="text" onInput={this.handlers.onFilterTextChange} className="form-control tableSearchInput" style={{ width:200 }}  />
-                        <span className="fa fa-search form-control-feedback" aria-hidden="true"></span>
+                        <span className="fa fa-search form-control-feedback" aria-hidden="true" style={{zIndex: 0}} />
```

Comment:
This is to prevent the table search box icon from overlaying the add tracks drop down menu when the menu is open:

![searchbox_zindex](https://user-images.githubusercontent.com/3604198/50786870-cb74f900-1282-11e9-8f4d-5a20fc6b51cd.png)

Setting the `zIndex` may not be the best way to solve it though.